modules/grafana/files/vhost.conf

```diff
@@ -8,10 +8,9 @@ server {
 
   access_log /var/log/nginx/grafana-access.log timed_combined;
   access_log /var/log/nginx/grafana-json.event.access.log json_event;
-  error_log /var/log/nginx/grafana-error.log;
+  error_log  /var/log/nginx/grafana-error.log;
```

Comment:
This space doesn't do anything right? Just making sure I'm not missing anything.